.circleci/config.yml

```diff
@@ -175,7 +175,7 @@ jobs:
         default: ""
     docker:
       - image: *GOLANG_IMAGE
-    resource_class: large
+    resource_class: xlarge
```

Comment:
Bumping this takes the key lint run part (for the main consul module) down from:

    INFO Execution took 2m12.841127973s 

to

    INFO Execution took 1m41.403607416s